Templates/HtmlSpoiler.tt

```diff
@@ -270,5 +305,15 @@
             }
         }
 	});
+
+    function toggleDarkLight() {
+	var body = document.getElementById("body");
+	var currentClassBody = body.className;
+	body.className = currentClassBody == "dark-mode" ? "light-mode" : "dark-mode";
+	var help = document.getElementById("help");
+	var currentClassHelp = help.className;
+	help.className = currentClassHelp == "dark-mode" ? "light-mode" : "dark-mode";
```

Comment:
This is not necessary if the css-selectors hit any `light-mode | dark-mode` classes appearing under body. Same reasoning why you don't apply the classes to every tag all over the place.